### PR TITLE
Clarify tokens & utils

### PR DESCRIPTION
**Pull request by tannerwelsh**
#### What does this PR do?

- Split out tokens from utils (borders > utils; spacing/shadows > tokens)
Using rubric: tokens define variables, utils define classes
- Change root navigation to mimic Sage DS (tokens, colors, typography
all top-level pages)
- Make new typography page and classes for headers, body text

#### Where should the reviewer start?

Screenshots show the new nav & page content

#### Any background context you want to provide?

In discussion w/ Amy today, we decided that we wanted to better clarify what a token is and what a util is, and create some examples of each.

Basically we borrowed what [Sage](https://sage-design-system.kajabi.com/pages/design/token) does and modified it slightly. Also took some inspiration from Bootstrap.

#### Screenshots (if appropriate)

Screen Shot 2021-03-08 at 4 34 04 PM
Screen Shot 2021-03-08 at 4 34 07 PM
Screen Shot 2021-03-08 at 4 34 11 PM
Screen Shot 2021-03-08 at 4 34 13 PM
Screen Shot 2021-03-08 at 4 33 17 PM

### CODE REVIEW

File: css/design_system/typography.scss [hunk 1]
```diff
@@ -0,0 +1,108 @@
+/**
+ * @title Heading 1
+ * @category typography
+ * @value .s-h1
+ * @order 1
```
Comment (kat3kasper) [inline css/design_system/typography.scss:5]: We shouldn't have to manually define the order here. They should be ordered by how they are written within the scss file
